### PR TITLE
Use a config map to configure bootstrapper.

### PR DESCRIPTION
**Pull request by jlewi**
* Use deployment manager to create a config map containing a YAML
  spec that will be used to configure the ksonnet app created by the
  bootstrapper.

Related to #802 Trigger bootstrapper with deployment manager

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/859)

### CODE REVIEW

File: docs/gke/configs/cluster.jinja [hunk 1]
```diff
@@ -352,8 +374,10 @@ e.g. creating namespaces, service accounts, stateful set to run the bootstrapper
           - name: kubeflow-bootstrapper
             image: {{ properties["bootstrapperImage"] }}
             workingDir: /opt/bootstrap
-            command: [ "/opt/kubeflow/bootstrapper"]
-            args: ["--in-cluster", "--namespace=kubeflow"]
+            command: 
+              - /opt/kubeflow/bootstrapper
+              - --in-cluster
+              - --namespace=kubeflow
```
Comment (kunmingg) [inline docs/gke/configs/cluster.jinja]: Bootstrapper need config location: 
--config=/etc/kubeflow/config.yaml

**Comment by jlewi** [comment]
Thanks @kunmingg PTAL

**Pull request merged**